packaging/linux/smoketest/vagrantcommon/generate-vagrantfiles.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+from collections import namedtuple
+import os
+
+template = """Vagrant.configure("2") do |config|
+  config.vm.box = "{}"
+  {}
+  config.vm.synced_folder "../vagrantcommon/", "/vagrant", type: "rsync"
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = "4096"
+  end
+  config.vm.provision "shell", path: "../vagrantcommon/provision_{}.bash"
+  config.ssh.forward_x11 = true
+  config.vm.network :forwarded_port, guest: 22, host: {}, id: "ssh"
+  config.vbguest.auto_update = false
+end
+"""
+
+Platform = namedtuple('Platform', ['name', 'box', 'type', 'port', 'extras'])
+
+platforms = [
+    # Platform('ubuntu-latest', 'ubuntu/eoan64', 'deb', 2401,
```

Comment:
i can't get this box to work right now. idk why but i'll ticket for later

Reply:
F

Reply:
e